components/automate-ui/src/app/page-components/projects-filter-dropdown/projects-filter-dropdown.spec.ts

```diff
@@ -120,15 +126,15 @@ describe('ProjectsFilterDropdownComponent', () => {
       expect(options.length).toEqual(2);
       options.forEach((option: HTMLInputElement, index: number) => {
         const { label, checked } = component.editableOptions[index];
-        expect(option.textContent).toEqual(label);
+        expect(option.textContent.trim()).toEqual(label);
```

Comment:
Why the need for the `trim` here? I would have expected that to happen elsewhere

Reply:
I did a shallow internet search and had thought that this was a common jasmine issue but you're right, this shouldn't need to be here.  I'll need to investigate what's going on and i'll re-request your review once i have a fix in.

Reply:
@blakestier - The culprit ended up being the way that I was formatting the these text elements in the HTML.   

While the formatting didn't change the look on the front end, it added return carriages that caused extra spaces to be rendered in the tests.  I've updated the html so that is no longer an issue and pulled the use of `trim()` from the tests.  Thanks for saying something!